services/compileGRPCStubs/nodejs/template/configTemplate.ts

```diff
@@ -5,7 +5,7 @@ export const configTemplate = `/**
  export default {
    web3Provider: process.env.PROVIDER_HOST,
    privateKey: process.env.PRIVATE_KEY,
-   signerPrivateKey: process.env.SIGNER_PRIVATE_KEY,
+   signerPrivateKey: process.env.PROVIDER_HOST,
```

Comment:
it should be `PRIVATE_KEY`, not the `PROVIDER_HOST`

Reply:
Fix in the latest commit